### PR TITLE
[TEST] - Added tests for FileDependency

### PR DESCRIPTION
**Pull request by pamval**
test only

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 18s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **71**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/cpf-plugin/job/PR-156/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 28s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **71**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/cpf-plugin/job/PR-156/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **71**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/cpf-plugin/job/PR-156/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **71**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/cpf-plugin/job/PR-156/6/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by pamval** [comment]
@pedrofvteixeira now for CPF :). Please review and squash merge.

File: core/src/test/java/pt/webdetails/cpf/packager/dependencies/FileDependencyTest.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+
+    public void testGetFileInputStream() throws IOException {
+        PathOrigin pathOrigin = mock (PathOrigin.class);
+        IReadAccess readAccess = mock (IReadAccess.class);
+        InputStream is = mock (InputStream.class);
+        when( pathOrigin.getReader(anyObject()) ).thenReturn(readAccess);
+
+        when (readAccess.getFileInputStream("/home/admin/t.js")).thenReturn(is);
```
Comment (pedrofvteixeira) [inline core/src/test/java/pt/webdetails/cpf/packager/dependencies/FileDependencyTest.java]: Checkstyle issues ( in various points in the class ).

File: core/src/test/java/pt/webdetails/cpf/packager/dependencies/FileDependencyTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
+package pt.webdetails.cpf.packager.dependencies;
```
Comment (pedrofvteixeira) [inline core/src/test/java/pt/webdetails/cpf/packager/dependencies/FileDependencyTest.java]: License header

**Comment by pamval** [comment]
nitpicker @pedrofvteixeira *test* file has been formatted although it didn't need to be.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1m 24s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **71**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/cpf-plugin/job/PR-156/7/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**